.env.example

```
@@ -19,4 +19,7 @@ SENTRY_DSN=https://your-key@o123456.ingest.sentry.io/7654321
 SENTRY_ENVIRONMENT=production
 SENTRY_TRACES_SAMPLE_RATE=0.1
 SENTRY_PROFILES_SAMPLE_RATE=0.1
-SENTRY_SEND_DEFAULT_PII=false
\ No newline at end of file
+SENTRY_SEND_DEFAULT_PII=false
+
+# Generate with: python -c "import secrets; print(secrets.token_hex(32))"
+ADMIN_API_TOKEN=your-generated-token-here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)